.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -82,7 +82,7 @@ This is a checklist for PR authors. Please make sure to complete all tasks and c
     - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
     - [ ] I verified that comments were added to code that is not self explanatory
     - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
-    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
```

Comment:
Why are we removing this line. Don't we also need to make sure that it is localized?

Reply:
I did not remove it, but I now realize that I missed adding `- [ ]` to the beginning

Reply:
Sorry, I'm confused. This line no longer exists: 
>I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)

Are you removing that on purpose?

Reply:
Sorry, I changed the wrong line 🤦 
Updated now